docs/integrating-with-linters.md

```diff
@@ -8,7 +8,6 @@ Linters usually contain not only code quality rules, but also stylistic rules. M
 Luckily it’s easy to turn off rules that conflict or are unnecessary with Prettier, by using these pre-made configs:
```

Comment:
Maybe this should tweak a little bit too.

Reply:
not sure im following. tweak to what?

Reply:
stylelint will no longer conflict with prettier starting from v15. that's why I only deleted the link for `stylelint-config-prettier`